src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__schoolmint_grow_observation_details.sql

```diff
@@ -487,6 +500,7 @@ with
             od.created,
             od.observed_at,
             od.observer_name,
+            cast(od.observer_employee_number as int64) as observer_employee_number,
```

Comment:
- [ ] move to original cte
- [ ] use `safe_cast`
- [ ] `int` should be sufficient